filters/openpolicyagent/internal/envoy/skipperadapter.go

```diff
@@ -0,0 +1,34 @@
+package envoy
+
+import (
+	"net/http"
+	"strings"
+
+	ext_authz_v3_core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	ext_authz_v3 "github.com/envoyproxy/go-control-plane/envoy/service/auth/v3"
+)
+
+func AdaptToEnvoyExtAuthRequest(req *http.Request, metadata *ext_authz_v3_core.Metadata, contextExtensions map[string]string) *ext_authz_v3.CheckRequest {
```

Comment:
maybe rename to `AdaptExtAuthRequest`, because otherwise we duplicate `envoy` while reading the caller code.